README.md

```diff
@@ -59,73 +59,73 @@ parameter to the `puppet_enterprise::profile::server` class with a string value
 
-```
-cd ~/ssl-build/puppet.example.com
-rpm2cpio puppet.example.com-puppet-client-1.0-1.noarch.rpm | cpio -idmv
-```
+  3. If the ssl-build folder contains RPM files for the host, find and extract the contents of the puppet-client rpm file:
 
-This creates a folder structure in the current directory beginning with `./etc/pki/katello-certs-tools/`
+            cd ~/ssl-build/puppet.example.com
+            rpm2cpio puppet.example.com-puppet-client-1.0-1.noarch.rpm | cpio -idmv
 
-d. Copy the `.crt` and `.key` files to your Puppet server, found either at:
+     This creates a folder structure in the current directory beginning with `./etc/pki/katello-certs-tools/`
 
-`~/ssl-build/puppet.example.com/puppet.example.com-puppet-client.crt`
+  4. Locate the `.crt` and `.key` files to your Puppet server. They are found either at:
 
-`~/ssl-build/puppet.example.com/puppet.example.com-puppet-client.key`
+            ~/ssl-build/puppet.example.com/puppet.example.com-puppet-client.crt
+            ~/ssl-build/puppet.example.com/puppet.example.com-puppet-client.key
 
-Or if you had to extract them from the RPM: 
-   `~/ssl-build/puppet.example.com/etc/pki/katello-certs-tools/certs/puppet.example.com-puppet-client.crt`
-     `~/ssl-build/puppet.example.com/etc/pki/katello-certs-tools/private/puppet.example.com-puppet-client.key`
+     Or if you had to extract them from the RPM:
+     
+            ~/ssl-build/puppet.example.com/etc/pki/katello-certs-tools/certs/puppet.example.com-puppet-client.crt
+            ~/ssl-build/puppet.example.com/etc/pki/katello-certs-tools/private/puppet.example.com-puppet-client.key
 
-Copy the files to `/etc/puppetlabs/puppet/ssl/satellite`.
+     Copy the files to `/etc/puppetlabs/puppet/ssl/satellite` on your Puppet server.
 
-e. On your Puppet server, set the ownership of these two files to `pe-puppet`:
+  5. On your Puppet server, set the ownership of these two files to `pe-puppet`:
 
+     Example (adjust paths and filenames accordingly):
 
-Example (adjust paths and filenames accordingly):
+            chown pe-puppet /etc/puppetlabs/puppet/ssl/satellite/puppet.example.com-puppet-client.crt
+            chown pe-puppet /etc/puppetlabs/puppet/ssl/satellite/puppet.example.com-puppet-client.key
 
-```
-chown pe-puppet /etc/puppetlabs/puppet/ssl/satellite/puppet.example.com-puppet-client.crt
-chown pe-puppet /etc/puppetlabs/puppet/ssl/satellite/puppet.example.com-puppet-client.key
-```
-
-f. In the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_smart_proxies` parameter to `true`. Additionally, add your Puppet server's FQDN to the `trusted_hosts` array on the same page; for example, `[puppet.example.com]`.
+  6. In the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_smart_proxies` parameter to `true`. Additionally, add your Puppet server's FQDN to the `trusted_hosts` array on the same page; for example, `[puppet.example.com]`.
```

Comment:
I'm not sure this combination makes sense. If you restrict it to smart proxies, the fact it's a smart proxy is sufficient. So this should be sufficient if the CN on the client certificate is also registered as a Smart Proxy:
```suggestion
  6. In the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_smart_proxies` parameter to `true`.
```

Reply:
See comment above; I do not mean to suggest content changes here, it's just a markdown formatting fix.